client/src/cordova/dev.action.mjs

```diff
@@ -134,7 +96,48 @@ export async function main(...givenParameters) {
     );
 
     return true;
-  }).listen(RELOAD_SERVER_PORT);
+  });
+
+  server.listen(0, 'localhost', async () => {
+    const serverAddress = `ws://localhost:${server.address().port}`;
```

Comment:
Localhost also gets mapped to an IP, which may matter in some corner cases. This is more robust:
```suggestion
    const serverAddress = `ws://${server.address().address}:${server.address().port}`;
```

Reply:
That's fine. `::1` is localhost in IPv6. The any address is `::`, without the 1.